code/modules/cargo/shipment_datums/guns.dm

```diff
@@ -0,0 +1,90 @@
+/datum/shipping/foam_force_shotguns
+	name = "FOAM FORCE(tm) Shotguns"
+	value = 126
+	sell_type = /obj/item/weapon/gun/projectile/shotgun/toy
+	amount_on_market = 200
+	category = "Weapons"
+
+/datum/shipping/flashbangs
+	name = "Flashbang Boxes"
+	value = 250
+	sell_type = /obj/item/weapon/storage/box/flashbangs
+	amount_on_market = 50
+	category = "Weapons"
+
+/datum/shipping/teargas_grenades
+	name = "Tear Gas Grenade Boxes"
+	value = 250
+	sell_type = /obj/item/weapon/storage/box/teargas
+	amount_on_market = 50
+	category = "Weapons"
+
+/datum/shipping/handcuff_boxes
+	name = "Handcuff Boxes"
+	value = 250
+	sell_type = /obj/item/weapon/storage/box/handcuffs
+	amount_on_market = 50
+	category = "Weapons"
+
+/datum/shipping/stun_batons
+	name = "Stun Batons"
+	value = 333
+	sell_type = /obj/item/weapon/melee/baton/loaded
+	amount_on_market = 50
+	category = "Weapons"
+
+/datum/shipping/lasers
+	name = "Laser Guns"
+	value = 500
+	sell_type = /obj/item/weapon/gun/energy/laser
+	amount_on_market = 20
+	category = "Weapons"
+
+/datum/shipping/tasers
+	name = "Tasers"
+	value = 500
+	sell_type = /obj/item/weapon/gun/energy/gun/advtaser
```

Comment:
This overlaps with the egun datum. I'm not sure what that would mean for export prices.
Also, the way this is done means there're a bunch of arbitrary restrictions on what guns you can and can't export. It would be better if you could export any guns, and the shipping price of the weapon was determined programatically, based on whether the gun is rechargeable, and the stun/damage/stamina damage/energy cost of each projectile it's capable of firing.
Import prices could be adjusted by just plugging in the numbers.